### DISCUSSION TITLE
Thanks for nebula

### DISCUSSION THREAD

**Discussion by chmac** [slackhq/nebula#1120] [General]
Apologies for the spurious issue. I just read [this](https://www.defined.net/blog/nebula-is-not-the-fastest-mesh-vpn/) and am now persuaded to stay with nebula rather than explore netmaker. Not because I care about throughput, I only have 5 machines connected over a home network with almost no traffic, but because of the article content. The attention to detail, compelling attempt to be as fair as possible, I found it persuasive that the folks at nebula are thinking hard about stuff.

Nebula has been a stable part of my very small network for several years now. It usually goes unnoticed, which is the whole point. Thanks for releasing it and maintaining it from one happy user.

**Comment by johnmaguire** [comment]
Hey Callum, thanks a lot for the praise! @rawdigits spent a lot of time and energy on getting performance testing right, so it's really nice to hear that it resonated and you found it valuable!

I moved this over to Discussions just to keep the issue tracker focused on issues, but thanks again for the positive feedback! :-)

**Reply by 1player** [reply]
Speaking of, I found a HN comment from one of your developers saying you spent a lot of time optimising the mobile app so that it does not drain battery and is as efficient as possible.

Given that Tailscale has been suffering from annoying mobile battery drain since forever (and the reason I looked for an alternative), I really believe you should advertise its mobile-friendliness as an always-on VPN much more. The only thing that has stopped me from setting up a true always-on mesh network is that I believed until a few minutes ago that no one had managed to create a VPN that I can keep running on my phone, with all my services at my fingertips. If it were not for that random comment on HN, I would never have known that Nebula was much ahead than Tailscale on that aspect.

You should be much more vocal about it, in my opinion! Thanks for the hard work.
